### PR TITLE
I18n: Add Chinese (Simplified)

### PR DESCRIPTION
**Pull request by joshhunt**
**What this PR does / why we need it**:

 - Adds Chinese (Simplified)
 - Adds documentation for what to update when adding a new locale

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/core/components/SharedPreferences/SharedPreferences.tsx [hunk 1]
```diff
@@ -43,15 +49,19 @@ const languages: Array<SelectableValue<string>> = [
...
+    label: t('common.locale.es-ES', 'Spanish'),
   },
   {
     value: FRENCH_FRANCE,
-    label: t('common.locale.fr', 'French'),
+    label: t('common.locale.fr-FR', 'French'),
+  },
+  {
+    value: CHINESE_SIMPLIFIED,
+    label: t('common.locale.cn-Hans', 'Chinese (Simplified)'),
```
Comment (JoaoSilvaGrafana) [inline public/app/core/components/SharedPreferences/SharedPreferences.tsx:64]: Is the difference between `cn-Hans` and `zh-Hans` what you were talking about earlier about the difference between the locales standards in Crowdin and i18next?

File: public/locales/en-US/grafana_old.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (JoaoSilvaGrafana) [inline public/locales/en-US/grafana_old.json:1]: Do we need to commit these? Wasn't this added to the `gitignore` in a different branch?

File: contribute/internationalization.md [hunk 3]
```diff
@@ -38,6 +38,18 @@ const ErrorMessage = ({ id, message }) => <Trans id={`errors.${id}`}>There was a
 
 3. Before submitting your PR, run the `yarn i18n:extract` command to extract the messages you added into the `messages.po` file and make them available for translation.
 
+## How to add a new language
```
Comment (JoaoSilvaGrafana) [inline contribute/internationalization.md:41]: 👏 👏 👏

**Review by JoaoSilvaGrafana — COMMENTED** [review]
Nice job adding it to the docs!

File: public/app/core/components/SharedPreferences/SharedPreferences.tsx (hunk 1 above)
Comment (joshhunt) [reply public/app/core/components/SharedPreferences/SharedPreferences.tsx:64]: Excellent spotting. I keep forgetting whether it's zh or ch!

**Review by osg-grafana — APPROVED** [review]

**Review by JoaoSilvaGrafana — APPROVED** [review]

**Pull request merged**